src/Features/Core/Portable/Shared/Utilities/SerializationConstructorCheck.cs

```diff
@@ -0,0 +1,28 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using Microsoft.CodeAnalysis.Shared.Extensions;
+
+namespace Microsoft.CodeAnalysis.Shared.Utilities
+{
+    internal readonly struct SerializationConstructorCheck
```

Comment:
Unfortunately I can use the GitHub suggested changes feature to rename this file, so even though the rest of the changes can be automated the rename will need to be done manually.